docs/2-advanced/07-accessibility.md

```diff
@@ -36,6 +36,15 @@ By doing so, you receive an input element with `role="combobox"` with all aria a
 In order to ease the setting of aria attributes, we have introduced properties that are available for developers to extend the accessibility support in the context of the application. More information about the available properties could be found in the Accessibility APIs section below.
 
 
+### **Invisible Messaging**
+The Invisible Message provides a way to programmaticaly expose dynamic content changes in a way that can be announced by screen readers. It marks the dynamic content changes as ARIA live regions so that we are able to inform the users of assistive technologies for a change that has happened to the UI.
+
+The Invisible Messaging service is designed to be used both internally in the components logic and from the applications. Using the service, you have to specify the message to be announced by the screen reader and the mode which will be inserted in the `aria-live` attribute via the `InvisibleMessage.announce(message, mode)` method. The possible modes to choose from are ` InvisibleMessageMode.Assertive` and  `InvisibleMessageMode.Polite`. `Assertive` indicates that updates to the region have the highest priority and should be presented to the user immediately. `Polite` indicates that updates to the region should be presented at the next graceful opportunity such as at the end of reading the current sentence, or when the user paused typing.
```

Comment:
I would do it like this:
> The possible modes to choose from are:

-  ` InvisibleMessageMode.Assertive`:  `Assertive` indicates that updates to the region have the highest priority and should be presented to the user immediately.
- `InvisibleMessageMode.Polite`.  `Polite` indicates that updates to the region should be presented at the next graceful opportunity such as at the end of reading the current sentence, or when the user paused typing.